specifications/barriers/README.md

```diff
@@ -0,0 +1,17 @@
+# Barrier synchronization
+-
+
+## Barriers.tla
+
+A specification of a Reusable Barrier synchronization facility using as
+presented in the INFO09012 Parallel Programming course in ULiège by 
+Prof. Pascal Fontaine.
+
+A refinement towards an abstract Barrier specification is proven with TLAPS.
+
+The main invariant `Inv` is verified in the provided model for $N = 7$ 
+processes.
```

Comment:
Barriers.tla proves other invariants as well, perhaps they could be included in the configuration? And the proof in Barriers.tla culminates in verifying refinement between the two specifications, TLC could also be used to check this for a finite instance.